mautic/mautic
# Fixed issue in Zoho that caused contacts to inherit values from previous contacts

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | y

[//]: # ( Required: )
#### Description:

There was a bug where if a contact had a blank value, it assumed the value of the previous contact that didn't. 

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Setup the zoho plugin and map required fields
2. Create several contacts in Mautic the last one without a last name
3. Sync with Zoho `php app/console m:i:s -i Zoho -a "1 day"`
4. Look in Zoho for the contact without the last name in Mautic
5. It'll have the last name of the previous contact

#### Steps to test this PR:
1. repeat the above; this time the contact without a last name should fail sync because last name is required; it should not inherit the previous contact's last name
2. Code review and run the new unit tests
